src/main/java/org/kairosdb/datastore/cassandra/cache/persistence/ElastiCacheConfiguration.java

```diff
@@ -0,0 +1,91 @@
+package org.kairosdb.datastore.cassandra.cache.persistence;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+
+public class ElastiCacheConfiguration {
+    private static final String PREFIX = "kairosdb.datastore.cassandra.cache.elasticache.";
+
+    private static final String HOSTNAME = PREFIX + "hostname";
+    private static final String PORT = PREFIX + "port";
+    private static final String CONNECTION_TIMEOUT_IN_MILLIS = PREFIX + "connection_timeout_in_millis";
+    private static final String WRITER_THREADS = PREFIX + "writer_threads";
+    private static final String WRITER_THREAD_IDLE_TIMEOUT_SECONDS = PREFIX + "writer_thread_idle_timeout_seconds";
+    private static final String TTL_IN_SECONDS = PREFIX + "ttl_in_seconds";
+    private static final String MAX_QUEUE_SIZE = PREFIX + "max_queue_size";
+    private static final String OPERATION_TIMEOUT_IN_MILLIS = PREFIX + "operation_timeout_in_millis";
+    private static final String CONNECTION_POOL_SIZE = PREFIX + "connection_pool_size";
+
+
+    @Inject(optional = true)
+    @Named(HOSTNAME)
+    private String hostName = "localhost";
+
+    @Inject(optional = true)
+    @Named(PORT)
+    private int port = 6379;
+
+    @Inject(optional = true)
+    @Named(CONNECTION_TIMEOUT_IN_MILLIS)
+    private int connectionTimeoutInMillis = 500;
+
+    @Inject(optional = true)
+    @Named(OPERATION_TIMEOUT_IN_MILLIS)
+    private int operationTimeoutInMillis = 500;
+
+    @Inject(optional = true)
+    @Named(WRITER_THREADS)
+    private int writerThreads = 20;
+
+    @Inject(optional = true)
+    @Named(WRITER_THREAD_IDLE_TIMEOUT_SECONDS)
+    private int workerThreadIdleTimeoutSeconds = 300;
+
+    @Inject(optional = true)
+    @Named(CONNECTION_POOL_SIZE)
+    private int connectionPoolSize = 50;
```

Comment:
If the writer threads are only 20 I don't think this needs to be 50. I'd change this to default to the same size as the writer threads